### ISSUE TITLE
register module is not public

### ISSUE THREAD

**Issue by tufelkinder** [NeoBirth/ADXL343.rs#37]
When I try to use the adxl343 module in my app and set register values, I receive the compile error that the adxl343::register module is not public. The Register enum is public, but it looks like the module needs to be as well.

**Comment by tarcieri** [comment]
Which type are you trying to access? `Register`?

If so, a PR would be accepted to re-export it from the toplevel.

**Comment by tufelkinder** [comment]
Correct: for example, I'm trying to call `adxl343.write_register(Register::INT_ENABLE, <value>);` and `write_register()` expects a Register Enum value as the first parameter, unless I'm mistaken.

If there is more to this than just redesignating `mod register` as `pub mod register` I'm probably not qualified...

**Comment by tarcieri** [comment]
You just need to add it to the re-exports here, and remove the private import:

https://github.com/NeoBirth/ADXL343.rs/blob/develop/src/lib.rs#L15
